code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -688,6 +688,22 @@
 		M.bodytemperature = min(330, M.bodytemperature + (20 * TEMPERATURE_DAMAGE_COEFFICIENT)) //310 is the normal bodytemp. 310.055
 	return ..()
 
+/datum/reagent/consumable/ethanol/adminfreeze
+	name = "Admin Freeze"
+	id = "adminfreeze"
+	description = "Ultimate Punishment."
+	reagent_state = LIQUID
+	color = "#30F0FF" // rgb: 048, 240, 255
+	alcohol_perc = 0.5
+	drink_icon = "adminfreeze"
+	drink_name = "Admin Freeze"
+	drink_desc = "The ultimate punishment."
+	taste_description = "a series of bad decisions...BWOINK"
+
+/datum/reagent/consumable/ethanol/adminfreeze/on_mob_life(mob/living/M)
```

Comment:
Won't this get triggered even if it gets into someone's body via syringe gun? The memechem is real...

Reply:
I guess it's like a 50% worse frost oil in absolute terms though... probably fine :sweat_smile:

Reply:
It's a direct copy of eating ice lol. I didn't want to end up killing people - unless they drink a shit ton, then it's their own fault :P

Reply:
Lol indeed.